browser/ui/views/frame/split_view_browsertest.cc

```diff
@@ -165,3 +166,36 @@ IN_PROC_BROWSER_TEST_F(SplitViewBrowserTest, SplitViewSizeDelta) {
   tab_strip_model().ActivateTabAt(3);
   EXPECT_EQ(kSizeDelta, contents_layout_manager->split_view_size_delta());
 }
+
+IN_PROC_BROWSER_TEST_F(SplitViewBrowserTest, SplitViewOrientation) {
+  // Given that we have a split view
+  brave::NewSplitViewForTab(browser());
+  browser()->tab_strip_model()->ActivateTabAt(0);
+  auto* browser_view = static_cast<BraveBrowserView*>(browser()->window());
+
+  // Then they should be laid out horizontally by default.
+  EXPECT_LT(browser_view->contents_web_view()->bounds().right(),
+            browser_view->secondary_contents_web_view()->x());
+  EXPECT_EQ(browser_view->contents_web_view()->y(),
+            browser_view->secondary_contents_web_view()->y());
+
+  // When toggle the orientation
+  brave::ToggleSplitViewOrientation(browser());
+
+  // Then they should be laid out vertically.
+  EXPECT_EQ(browser_view->contents_web_view()->x(),
+            browser_view->secondary_contents_web_view()->x());
+  EXPECT_LT(browser_view->contents_web_view()->bounds().bottom(),
+            browser_view->secondary_contents_web_view()->y());
+
+  // When creating new split view,
+  chrome::AddTabAt(browser(), GURL(), -1, /*foreground*/ true);
+  brave::NewSplitViewForTab(browser());
+  browser()->tab_strip_model()->ActivateTabAt(2);
```

Comment:
Why need this activation?
`chrome::AddTabAt()` create NTP and it's activated(foreground) at `2`
and `brave::NewSplitViewForTab()` will pick that active tab(at 2).
So, I expected new tile will have horizonal layout w/o this explicit activation.
Am I missing something?

Reply:
I thought at(3) would be the active tab. Removed.

Reply:
Test failed. brave://NewSplitView add a new tab and activate it. So at(3) seems to be the active one.

Reply:
Ah, ok. How about passing NTP tab at 2 to `brave::NewSplitViewForTab()` for covering https://github.com/brave/brave-core/pull/24257/commits/e9e011c677c342d2230f537559cc7fe2d94fc55d?
Then maybe this activation is not needed?

Reply:
Hmm, it seems `brave::NewSplitViewForTab()` always create new tab..

Reply:
when tab at(3) is the active one, main contents will be after secondary contents, which makes test complex, so I activated tab at 3.

Reply:
ok